packages/components-library/src/components/CartSelectionToast.vue

```diff
@@ -5,6 +5,9 @@
     no-auto-hide
     :toaster="location"
     :variant="backgroundVariant"
+    :toastClass="toastClass"
```

Comment:
:S really don't like the way this scales, new prop for every class detail , maybe use a slot instead ?

Reply:
This something that a b-toast has, I only expose it in our wrapper. nothing more nothing less. Has nothing to do with scalability